### PR TITLE
docs: guide example page translated into Korean

### PR DESCRIPTION
**Pull request by Gaic4o**
I have translated the Guide page into Korean.

If you think there are aspects that need improvement, please let me know, and I would be happy to make the adjustments. Thank you!

I have requested a Korean translation review from one reviewer in the FSD Discord Korean community to ensure that my Korean writing is well done. I would appreciate it if you could merge it after the review is completed.

### CODE REVIEW

**Comment by illright** [comment]
Thanks! Could you also ask someone from Discord to check these translations?

**Comment by Gaic4o** [comment]
@illright  Got it, thank you.

File: i18n/kr/docusaurus-plugin-content-docs/current/guides/examples/index.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<NavCard
+    title="Types"
+    description="타입 위치 선정 및 FSD에서의 타입 종류"
+    to="/docs/guides/examples/types"
+    Icon={FontSizeOutlined}
+    tags={["DTO", "Mappers", "Entity relationships", "Auto-generation", "Validation schemas"]}
+/>
+<NavCard
+    title="Page layouts"
+    description="FSD에서 타입을 어디에 배치하고 활용할지에 대한 가이드"
```
Comment (movie42) [inline i18n/kr/docusaurus-plugin-content-docs/current/guides/examples/index.mdx:32]: page layouts을 직접 번역하셨기 때문에 아시겠지만 해당 description이 Types discription의 내용이 섞여 들어간것 같습니다. 원문 내용 확인하셔서 수정해주세요.

**Review by movie42 — CHANGES_REQUESTED** [review]

**Review by movie42 — CHANGES_REQUESTED** [review]
코멘트 확인부탁드립니다.

**Comment by Gaic4o** [comment]
@movie42  제가 타입쪽 `description`을 작성한 것 같네요. `title`에 맞게 다시 `description`을 수정했어요!

**Review by movie42 — APPROVED** [review]
고생하셨습니다! 👍

**Review by illright — APPROVED** [review]

**Pull request merged**